modules/qualitycontrol/CodecSelection.js

```diff
@@ -72,11 +72,10 @@ export class CodecSelection {
             }
 
             // Push VP9 to the end of the list so that the client continues to decode VP9 even if its not
-            // preferable to encode VP9 (because of browser bugs on the encoding side or added complexity on mobile
-            // devices). Currently, VP9 encode is supported on Chrome and on Safari (only for p2p).
+            // preferable to encode VP9 (because of browser bugs on the encoding side or other reasons).
             const isVp9EncodeSupported = browser.supportsVP9() || (browser.isWebKitBased() && connectionType === 'p2p');
 
-            if (!isVp9EncodeSupported || this.conference.isE2EEEnabled()) {
```

Comment:
Doesn't the E2EE part still stand?

Reply:
It gets enforced [here](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/qualitycontrol/CodecSelection.js#L161) when the codec is selected so the check here is redundant.